### ISSUE TITLE
submission question

### ISSUE THREAD

**Issue by alchan00** [SNU-ARC/2024_spring_sysprog_Lab2#9]
I followed the instruction and codes in github read me.

$ ls
2024-12345.pdf  Makefile  README.md  reference  src  tools
$ tar -cvf 2024-12345.tar src/dirtree.c 2024-12345.pdf
src/dirtree.c
2024-12345.pdf
$ file 2024-12345.tar
2024-12345.tar: POSIX tar archive (GNU)

If we just copy the code we compress src as well. i.e. src(which contains dirtree.c) and ~.pdf
Is it correct that in the tar file there is a folder src and dirtree.c in it?

**Comment by yunjayh** [comment]
Yes. there will be a src directory with your solution and your report in the tar file. 

Please make sure that your report and the tar file are named as your student id, not `2024-12345` 😉
